### DISCUSSION TITLE
Any (in)significant reason(s) to use 172.17.17.2, besides fewer conflicts with WAN side non-bridged routers?

### DISCUSSION THREAD

**Discussion by webmasterkai** [SmoothWAN/SmoothWAN#97] [General & Help]
I see in Release 0.99.5 in the default LAN IP was [changed](https://github.com/SmoothWAN/SmoothWAN/commit/d5c5b59abdf54ba7fa8f7f7f92b884e538c62168) from `192.168.3.1` to `172.17.17.2`. The [release note](https://github.com/SmoothWAN/SmoothWAN/releases/tag/0.99.5) mentions changing the "default IP Changed default LAN to a ~unique address".

I'm curious if there is any reason besides fewer conflicts with WAN side devices?

Thanks!

**Comment by TalalMash** [comment]
Few reasons if I remember correctly:  DNS/DHCP renewal delay when changing between different network with the same IP and SSID during tests, and being able to access the Web UI on WAN side devices that commonly use the 192.168.x.x format and vice versa when used as WAN.
It is configuration compatibility address, there are no issues changing it if you have configured the WAN side.
